IWXXM/sigmet.xsd

```diff
@@ -1,497 +1,407 @@
-						<annotation>
-							<documentation>The volcano that is erupting - required in all cases other than cancellations</documentation>
-						</annotation>
-					</element>
-					<element name="volcanicAshMovedToFIR" type="iwxxm:AirspacePropertyType" minOccurs="0" maxOccurs="1">
-						<annotation>
-							<documentation>The FIR to which volcanic ash has moved.  Reported only on SIGMET cancellation reports</documentation>
-						</annotation>
-					</element>
-				</sequence>
-			</extension>
-		</complexContent>
-	</complexType>
-	<complexType name="VolcanicAshSIGMETPropertyType">
-		<sequence minOccurs="0">
-			<element ref="iwxxm:VolcanicAshSIGMET"></element>
-		</sequence>
-		<attributeGroup ref="gml:AssociationAttributeGroup"></attributeGroup>
-		<attributeGroup ref="gml:OwnershipAttributeGroup"></attributeGroup>
-	</complexType>
-	<element name="SIGMETPositionCollection" type="iwxxm:SIGMETPositionCollectionType" substitutionGroup="gml:AbstractFeature">
-		<annotation>
-			<documentation>A collection of SIGMETPositions, each representing a location where meteorological conditions exist.  All members of this collection are of type SIGMETPosition.</documentation>
-		</annotation>
-	</element>
-	<complexType name="SIGMETPositionCollectionType">
-		<complexContent>
-			<extension base="gml:AbstractFeatureType">
-				<sequence>
-					<element name="phenomenonTime" type="iwxxm:AbstractTimeObjectPropertyType">
-						<annotation>
-							<documentation>The time at which meteorological phenomena occur</documentation>
-						</annotation>
-					</element>
-					<element nillable="true" name="member" type="iwxxm:SIGMETPositionPropertyType" minOccurs="1" maxOccurs="unbounded">
-						<annotation>
-							<documentation>The members of this collection</documentation>
-						</annotation>
-					</element>
-					<element name="extension" type="iwxxm:ExtensionType" minOccurs="0" maxOccurs="unbounded">
-						<annotation>
-							<documentation>Extension block for optional and/or additional parameters for element SIGMETPositionCollection</documentation>
-						</annotation>
-					</element>
-				</sequence>
-			</extension>
-		</complexContent>
-	</complexType>
-	<complexType name="SIGMETPositionCollectionPropertyType">
-		<sequence minOccurs="0">
-			<element ref="iwxxm:SIGMETPositionCollection"></element>
-		</sequence>
-		<attributeGroup ref="gml:AssociationAttributeGroup"></attributeGroup>
-		<attributeGroup ref="gml:OwnershipAttributeGroup"></attributeGroup>
-	</complexType>
-	<simpleType name="SIGMETExpectedIntensityChangeType">
-		<annotation>
-			<documentation>Expected change in intensity for (significant) meteorological phenomena.
+The expected end of occurrence of volcanic ash ("NO VA EXP") is indicated with a missing SIGMET phenomenon with a nil reason of nothingOfOperationalSignificance</xs:documentation>
+						</xs:annotation>
+					</xs:element>
+					<xs:element name="analysisCollection" minOccurs="0" maxOccurs="1">
```

Comment:
The maxOccurs should be set to maxOccurs="unbounded" to allow reporting multiple VA clouds. More than one occurrence will be used only by VA SIGMET.

Reply:
Both updated in commit https://github.com/wmo-im/iwxxm/commit/14b70a04a0e06106377cd927d6dfecf9ca62f08f